W-11501748-August2RNs-ESB (#305)
mulesoft/docs-composer: en/mulesoft/modules/ROOT/pages/ms_composer_ms_release_notes.adoc

* In Salesforce Connector, you can now specify conditions for triggers.

> * The following Record Types are now available for NetSuite Connector:

Reviewer: it's probably fine, but confirm caps on Record Types

> * The following record yypes are now available for NetSuite Connector:

Reviewer: ```suggestion
* The following record types are now available for NetSuite Connector:
```

 ** SERVICE_RESALE_ITEM
 ** TIME_SHEET
 ** TIME_BILL

include::_partials/release-notes/rn-headings.adoc[tag=fixedIssues,leveloffset=+3]

[%header%autowidth.spread]

|===

|Issue |ID

> |Fixed the issue in ServiceNow Connector in which some labels were not properly capitalized. |W-10575461

Reviewer: ```suggestion
|Labels are now properly capitalized in ServiceNow Connector. |W-10575461
```

> |Fixed the issue in NetSuite Connector in which connections using lowercase values in the `Account ID` field failed. |W-11469671

Reviewer: ```suggestion
|In NetSuite Connector, connections using lowercase values in the `Account ID` field no longer fail. |W-11469671
```
